README.rst

```diff
@@ -110,13 +110,17 @@ own instance of Open edX, or released to the wider community).
 .. _XBlock review guidelines: https://openedx.atlassian.net/wiki/display/OPEN/XBlock+review+guidelines
 
 
-External XBlocks
+Example XBlocks
 ----------------
```

Comment:
A minor nit, but please keep the punctuation line the same length as the section title. `</ocd>`